### PR TITLE
add v6.1.2 release notes

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11678
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.2.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        - (dup) Fix the issue that DM reports the `Specified key was too long` error [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) Fix the issue that latin1 data might be corrupted during replication [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
+    + TiCDC **owner: @nongfushanquan**
+
+        - Fix a bug that may cause cdc server panic if it received a http request before cdc server fully started [#6838](https://github.com/pingcap/tiflow/issues/6838)
+        - Change log level from info to debug for some logs to avoid too many logs [#7235](https://github.com/pingcap/tiflow/issues/7235)
+        - Fix a bug that may cause changefeed's redo log files be deleted wrongly [#6413](https://github.com/pingcap/tiflow/issues/6413)
+        - Fix a bug that may cause cdc unavaliable by commit too many operation in a etcd transaction [#7131](https://github.com/pingcap/tiflow/issues/7131)
+
```
Comment (nongfushanquan) [inline releases/release-6.1.2.md:121]: -fix a bug which can lead inconsistency Change if non-reentrant DDLs can be executed twice  [#6927]https://github.com/pingcap/tiflow/issues/6927
-enhance the region worker's performance by  handling the resolved ts in the batch mode [#7078]https://github.com/pingcap/tiflow/issues/7078

**Review by nongfushanquan — APPROVED** [review]

**Review by fixdb — APPROVED** [review]
+1

File: releases/release-6.1.2.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
++ TiFlash
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - (dup) Fix the issue that a window function might cause TiFlash to crash when the query is canceled [#5814](https://github.com/pingcap/tiflash/issues/5814)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - (dup) Fix the panic that occurs after creating the primary index with a column containing the `NULL` value [#5859](https://github.com/pingcap/tiflash/issues/5859)
```
Comment (flowbehappy) [inline releases/release-6.1.2.md:101]:
```suggestion
    - (dup) Fix the panic that occurs after creating the primary index with a column containing the `NULL` value [#5859](https://github.com/pingcap/tiflash/issues/5859)
    - Fix the issue that iolimiter could unexpectedly throttle io throughput of query requests after large volume writes, which slows down query performance. [#5801](https://github.com/pingcap/tiflash/issues/5801)
    - 
```

File: releases/release-6.1.2.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    - Fix the issue that GROUP CONCAT with ORDER BY might fail when the ORDER BY clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
+    - (dup) Fix the issue that `Can't find column` is reported if an `UPDATE` statement contains common table expressions (CTE) [#35758](https://github.com/pingcap/tidb/issues/35758)
+    - (dup) Fix the issue that the `EXECUTE` might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
+
++ TiKV **owner: @ethercflow**
+
+    - (dup) Fix the issue of QPS drop when flow control is enabled and `level0_slowdown_trigger` is set explicitly [#11424](https://github.com/tikv/tikv/issues/11424)
+    - (dup) Fix the issue that causes permission denied error when TiKV gets an error from the web identity provider and fails back to the default provider [#13122](https://github.com/tikv/tikv/issues/13122)
+    - (dup) Fix the issue that the TiKV service is unavailable for several minutes when a TiKV instance is in an isolated network environment [#12966](https://github.com/tikv/tikv/issues/12966)
+
```
Comment (ethercflow) [inline releases/release-6.1.2.md:85]:
```suggestion
- Fix the issue that snapshot may not contain the complete data caused by introduced batch snapshot across regions. [#13553](https://github.com/tikv/tikv/issues/13553)
```

File: releases/release-6.1.2.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
++ TiDB
+
+    <!--sql-infra **owner: @wjhuang2016**-->
+
+    - Allow setting Placement Rules and TiFlash replica at the same time in one table [#37171](https://github.com/pingcap/tidb/issues/37171)
+
+    <!--execution **owner: @zanmato1984**-->
+
+    - (dup) Fix the wrong result that occurs when enabling dynamic mode in partitioned tables for TiFlash [#37254](https://github.com/pingcap/tidb/issues/37254)
```
Comment (XuHuaiyu) [inline releases/release-6.1.2.md:23]: Move this to the **bug fixes** section.
Rest LGTM

File: releases/release-6.1.2.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    - (dup) Fix the issue that a window function might cause TiFlash to crash when the query is canceled [#5814](https://github.com/pingcap/tiflash/issues/5814)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - Fix the issue that iolimiter could unexpectedly throttle io throughput of query requests after large volume writes, which slows down query performance [#5801](https://github.com/pingcap/tiflash/issues/5801)
+    - (dup) Fix the panic that occurs after creating the primary index with a column containing the `NULL` value [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
```
Comment (niubell) [inline releases/release-6.1.2.md:106]: Fix the issuse of panic on invalid mertic countter operation [#37338](https://github.com/pingcap/tidb/issues/37338)

File: releases/release-6.1.2.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
++ TiFlash
+
+<!--compute **owner: @zanmato1984**-->
+
+<!--storage **owner: @flowbehappy**-->
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
```
Comment (niubell) [inline releases/release-6.1.2.md:42]: add retryable driver error on checksum [#37690](https://github.com/pingcap/tidb/issues/37690)

**Review by ethercflow — APPROVED** [review]
LGTM

File: releases/release-6.1.2.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - (dup) Fix the issue that DM reports the `Specified key was too long` error [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) Fix the issue that latin1 data might be corrupted during replication [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
```
Comment (niubell) [inline releases/release-6.1.2.md:115]: Fix the issue dm-worker hangs after running dm tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)

**Review by nolouch — APPROVED** [review]

File: releases/release-6.1.2.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+    + TiCDC **owner: @nongfushanquan TW: @hfxsd**
+
+        - Fix a bug that may cause cdc server panic if it received a http request before cdc server fully started [#6838](https://github.com/pingcap/tiflow/issues/6838)
+        - Change log level from info to debug for some logs to avoid too many logs [#7235](https://github.com/pingcap/tiflow/issues/7235)
+        - Fix a bug that may cause changefeed's redo log files be deleted wrongly [#6413](https://github.com/pingcap/tiflow/issues/6413)
+        - Fix a bug that may cause cdc unavaliable by commit too many operation in a etcd transaction [#7131](https://github.com/pingcap/tiflow/issues/7131)
+        - Fix a bug which can lead inconsistency Change if non-reentrant DDLs can be executed twice [#6927](https://github.com/pingcap/tiflow/issues/6927)
+        - Enhance the region worker's performance by handling the resolved ts in the batch mode [#7078](https://github.com/pingcap/tiflow/issues/7078)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:127]:
```suggestion
        - Fix a bug that the cdc server might panic if it receives an HTTP request before the cdc server fully starts [#6838](https://github.com/pingcap/tiflow/issues/6838)
        - Fix log flooding issue in TiCDC check [#7235](https://github.com/pingcap/tiflow/issues/7235)
        - Fix a bug that changefeed's redo log files might be deleted by mistake [#6413](https://github.com/pingcap/tiflow/issues/6413)
        - Fix a bug that cdc might become unavaliable when too many operations in an etcd transaction are committed  [#7131](https://github.com/pingcap/tiflow/issues/7131)
        - Fix a bug that data inconsistency might occur when non-reentrant DDL statemens in redo logs are executed twice [#6927](https://github.com/pingcap/tiflow/issues/6927)
        - Enhance the performance of the region worker by handling the resolved ts in batch [#7078](https://github.com/pingcap/tiflow/issues/7078)
```

File: releases/release-6.1.2.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+    <!--transaction **owner: @cfzjywxk**-->
+
+    - Fix the issue that the Region cache is not cleaned up in time when the Region is merged [#37141](https://github.com/pingcap/tidb/issues/37141)
+    - Fix the issue that the KV client sends unnecessary ping messages [#36861](https://github.com/pingcap/tidb/issues/36861)
+    - (dup) Fix the issue that the `EXPLAIN ANALYZE` statement with DML executors might return result before the transaction commit finishes [#37373](https://github.com/pingcap/tidb/issues/37373)
+
+    <!--planner **owner: @fixdb**-->
+
+    - Fix the issue that GROUP CONCAT with ORDER BY might fail when the ORDER BY clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
```
Comment (TomShawn) [inline releases/release-6.1.2.md:77]:
```suggestion
    - Fix the issue that `GROUP CONCAT` with `ORDER BY` might fail when the `ORDER BY` clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
```

File: releases/release-6.1.2.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+
++ TiFlash **TW: @Oreoxmt**
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - (dup) Fix the issue that a window function might cause TiFlash to crash when the query is canceled [#5814](https://github.com/pingcap/tiflash/issues/5814)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - Fix the issue that I/O Limiter might incorrectly throttle the IO throughput of query requests after bulk writes, which reduces the query performance [#5801](https://github.com/pingcap/tiflash/issues/5801)
```
Comment (TomShawn) [inline releases/release-6.1.2.md:103]:
```suggestion
    - Fix the issue that I/O Limiter might incorrectly throttle the I/O throughput of query requests after bulk writes, which reduces the query performance [#5801](https://github.com/pingcap/tiflash/issues/5801)
```

File: releases/release-6.1.2.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+    + TiDB Lightning **owner: @niubell TW: @shichun-0415**
+
+        - Fix the issue of panic on invalid mertic countter operation [#37338](https://github.com/pingcap/tidb/issues/37338)
+
+    + TiDB Data Migration (DM) **owner: @niubell TW: @shichun-0415**
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010), [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - Fix the issue dm-worker hangs after running dm tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)
```
Comment (shichun-0415) [inline releases/release-6.1.2.md:116]:
```suggestion
        - Fix the issue that upstream table structure information is lost when DM tasks enter the sync unit and are interrupted [#7159](https://github.com/pingcap/tiflow/issues/7159)
        - Fix large transaction errors by spliting SQL statements when saving checkpoints [#5010](https://github.com/pingcap/tiflow/issues/5010)
        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
        - Fix the issue dm-worker hangs after running dm tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)
```

File: releases/release-6.1.2.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+    + TiDB Lightning **owner: @niubell TW: @shichun-0415**
+
+        - Fix panic of TiDB Lightning caused by invalid metric counters [#37338](https://github.com/pingcap/tidb/issues/37338)
+
+    + TiDB Data Migration (DM) **owner: @niubell TW: @shichun-0415**
+
+        - Fix the issue that upstream table structure information is lost when DM tasks enter the sync unit and are interrupted [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - Fix large transaction errors by spliting SQL statements when saving checkpoints [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - Fix the issue that DM precheck requires the `SELECT` privilege on `INFORMATION_SCHEMA` [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - Fix the issue dm-worker triggers a deadlock error after running dm tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:117]:
```suggestion
        - Fix the issue that upstream table structure information is lost when DM tasks enter the sync unit and are interrupted [#7159](https://github.com/pingcap/tiflow/issues/7159)
        - Fix large transaction errors by splitting SQL statements when saving checkpoints [#5010](https://github.com/pingcap/tiflow/issues/5010)
        - Fix the issue that DM precheck requires the `SELECT` privilege on `INFORMATION_SCHEMA` [#7317](https://github.com/pingcap/tiflow/issues/7317)
        - Fix the issue that dm-worker triggers a deadlock error after running DM tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by flowbehappy — APPROVED** [review]

**Review by XuHuaiyu — APPROVED** [review]

**Review by niubell — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by wjhuang2016 — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**